RollbarSwift/Sources/RollbarSwift/RollbarExceptionGuard.m

```diff
@@ -0,0 +1,101 @@
+//
+//  RollbarExceptionGuard.m
+//  
+//
+//  Created by Andrey Kornich on 2021-03-04.
+//
+
+#import "RollbarExceptionGuard.h"
+#import "RollbarTryCatch.h"
+
+@implementation RollbarExceptionGuard {
+@private
+    RollbarLogger *logger;
+}
+
+-(BOOL)tryExecute:(nonnull void(NS_NOESCAPE^)(void))block {
+    NSError *error = nil;
+    return [self execute:block error:&error];
+}
+
+-(BOOL)execute:(nonnull void(NS_NOESCAPE^)(void))tryBlock
+            error:(__autoreleasing NSError * _Nullable * _Nullable)error {
+    
+    __block BOOL success = NO;
+    __block NSError* exceptionError = nil;
+    
+    [RollbarTryCatch try:^(void) {
+            
+            tryBlock();
+            success = YES;
+        }
+                       catch:^(NSException *exception) {
+            
+            exceptionError = [self convertException:exception];
+            
```

Comment:
Nit: What's up with the indentation here? Is this a generated file, or can we fix this up?

Reply:
just replied Xcode default indentation.